tests/json-fixtures/test_state.py

```diff
@@ -144,6 +146,11 @@ def expand_fixtures_forks(all_fixtures):
     # The result is in conflict with the yellow-paper:
     # * https://github.com/ethereum/py-evm/pull/1224#issuecomment-418800369
     ('stRevertTest/RevertInCreateInInit.json', 'RevertInCreateInInit', 'Byzantium', 0),
+    ('stRevertTest/RevertInCreateInInit.json', 'RevertInCreateInInit', 'Constantinople', 0),
+    # The CREATE2 variant seems to have been derived from the one above - it, too,
+    # has a "synthetic" state, on which py-evm flips.
+    # * https://github.com/ethereum/py-evm/pull/1181#issuecomment-446330609
+    ('stCreate2/RevertInCreateInInitCreate2.json', 'RevertInCreateInInitCreate2', 'Constantinople', 0),  # noqa: E501
```

Comment:
This change is made obsolete by PR #1577. Will rebase and rework into `BlockchainTests` when that is merged.